include/groonga/groonga.h

```diff
@@ -1693,6 +1693,24 @@ grn_column_get_all_index_columns(grn_ctx *ctx,
                                  grn_obj *column,
                                  grn_obj *index_columns);
 
+/**
+ * \brief Delete an object (such as a table or column) identified by the ID from
+ *        the database.
+ *
+ * \note This is an internal API intended for use by mroonga.
```

Comment:
```suggestion
 * \note This is an internal API intended for use by Mroonga.
```

ツールやライブラリの表記にはみんな細かい！

Reply:
やってしまいましたmm